staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
+  - single-numa-node
+  - isolate-thread-policy
+zones:
+  node-0:
+    type: Node
+    resources:
+      cpu:
+        capacity: 20
+        allocatable: 10
+      vendor/nic1:
+        capacity: 3
+        allocatable: 3
+  node-1:
+    type: Node
```

Comment:
where are types enumerated and described?

Reply:
In this document https://docs.google.com/document/d/12kj3fK8boNuPNqob6F_pPU9ZTaNEnPGaXEooW1Cilwg/edit section 4. Definitions of Keys, honestly it not so descriptive.

Reply:
It's really not :)  Can we do better?

Reply:
yes, will do )

Reply:
The  Zone types have been enumerated [here](https://github.com/kubernetes/kubernetes/pull/96275/commits/de4a1c63e96ba2935e3215afcc24ba761bf7e123#diff-e3cf903b50e93129e020599f1f7d2b9af992794ec97a02227433a8887a25184fR44-R54).